plugins/vpc-branch-eni/config/netconfig.go

```diff
@@ -37,6 +37,8 @@ type NetConfig struct {
 	BranchGatewayIPAddress net.IP
 	InterfaceType          string
 	UserName               string
+	Uid                    string
```

Comment:
I think these should be UID and GID to fit the golang variable naming convention. Also their types should be int, not string. NetConfigJSON contains the fields as they appear in the JSON file. NetConfig contains the parsed fields with correct types after validation. For example, see BranchIPAddress. It is a string in netConfigJSON but net.IPNet in NetConfig.

Reply:
I am following convention here: https://golang.org/pkg/os/user/, if you really like `UID` I can do that as well.

As for type, my concern about using int is we can't really tell if the UID is set or not, as by default the int is set to 0, and you don't know if it's unset or it's set and set to 0. We can set it to -1 to indicate the UID is not set, but to me it's lets preferred as it introduce some implicit assumptions.

Reply:
Is there any case where UID would not be set when using TAP mode? It seems to me that UID should always be set, hence the value 0 is not ambiguous.